content/blog/multi-lang-components/index.md

```diff
@@ -14,29 +14,31 @@ tags:
     - typescript
 ---
 
-Today we're excited to announce support for one of our most highly-requested features: multi-language components! Our
-first multi-language component is the popular [EKS package], which was previously only available for TypeScript. With the
-[v0.22.0 release], teams can [manage EKS clusters] in their language of choice (Python, .NET, Go, and TypeScript).
+Pulumi's infrastructure as code tooling combines the programming languages and tools you already know with the full power of cloud
+infrastructure. But, until now, some of that Pulumi components for that cloud infrastructure, like our popular [EKS package], were
```

Comment:
infrastructure. But, until now, some  Pulumi components for cloud infrastructure, like our popular [EKS package], were